lib/geocoder/providers/google_maps.ex

```diff
@@ -101,14 +103,26 @@ defmodule Geocoder.Providers.GoogleMaps do
 
   defp request_all(path, params) do
     httpoison_options = Application.get_env(:geocoder, Geocoder.Worker)[:httpoison_options] || []
-    get(path, [], Keyword.merge(httpoison_options, params: Enum.into(params, %{})))
+    resp = get(path, [], Keyword.merge(httpoison_options, params: Enum.into(params, %{})))
     |> fmap(&Map.get(&1, :body))
-    |> fmap(&Map.get(&1, "results"))
+
+    case resp do
```

Comment:
Can you make this a private function? That way we can pipe the return value from `fmap` in line 107 into it.